test/integration/matchers/matcher-utils.ts

```diff
@@ -182,8 +182,9 @@ export function buildReceivedResolvedModelCellStyle(cell: mxCell, bv = bpmnVisua
   return toBpmnStyle(bv.graph.getCellStyle(cell), cell.edge);
 }
 
+/* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
 function toBpmnStyle(rawStyle: StyleMap, isEdge: boolean): BpmnCellStyle {
-  const style: BpmnCellStyle = {
+  return {
```

Comment:
**suggestion:** can we add a `as BpmnCellStyle` to the return value. It will provide guidance in IDE?